core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -623,27 +623,31 @@ public void resetState() {
   public Map<String, Boolean> updateConfiguration(Map<String, String> propertiesMap) {
     Map<String, Boolean> result = new HashMap<>();
     int successCount = 0;
+    if (!Configuration.getBoolean(PropertyKey.CONF_DYNAMIC_UPDATE_ENABLED)) {
+      LOG.warn("Dynamic configuration update disabled but attempted, set {} to true to enable it",
+          PropertyKey.CONF_DYNAMIC_UPDATE_ENABLED.getName());
+      return result;
```

Comment:
what does this empty return value mean? I thought this is `<key, boolean=succeeded>`?